src/Mvc/Mvc.Core/test/Formatters/SystemTextJsonInputFormatterTest.cs

```diff
@@ -1,40 +1,80 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
+using System.Text;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
 using Xunit;
 
 namespace Microsoft.AspNetCore.Mvc.Formatters
 {
     public class SystemTextJsonInputFormatterTest : JsonInputFormatterTestBase
     {
-        [Fact(Skip = "https://github.com/aspnet/AspNetCore/issues/8474")]
+        [Fact]
```

Comment:
While it's generally not useful to have these methods here, it's way more useful to be able to have overriden methods for debugging. In the absence of this you have to use Test Explorer to be able debug. I'm ok with leaving this in.